script/install.sh

```diff
@@ -22,14 +22,21 @@ workDir=`cd ${workDir};pwd`
 source ${workDir}/env/install_env.sh
 source ${workDir}/env/dolphinscheduler_env.sh
 
+
 echo "1.create directory"
 
 # If install Path equal to "/" or related path is "/" or is empty, will cause directory "/bin" be overwrite or file adding,
 # so we should check its value. Here use command `realpath` to get the related path, and it will skip if your shell env
 # without command `realpath`.
 if [ ! -d $installPath ];then
-  sudo mkdir -p $installPath
-  sudo chown -R $deployUser:$deployUser $installPath
+  echo "sudo enabled = ${sudoEnable}"
+   if [ ${sudoEnable} = true ]; then
+      sudo mkdir -p $installPath
+      sudo chown -R $deployUser:$deployUser $installPath
+    else
+      mkdir -p $installPath
+      chown -R $deployUser:$deployUser $installPath
+    fi
```

Comment:
Why we need this?  should we directly remove the `sudo` and require user have permission of `installPath`?